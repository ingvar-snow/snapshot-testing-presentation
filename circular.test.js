


let obj1 = {};
let obj2 = {
	pointerToObj1: obj1
}
obj1.pointerToObj2 = obj2;


test('trying to break jest', () => {

  expect(obj2).toMatchSnapshot();
  
});



