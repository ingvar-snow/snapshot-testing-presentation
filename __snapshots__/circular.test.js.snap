// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`trying to break jest 1`] = `
Object {
  "pointerToObj1": Object {
    "pointerToObj2": [Circular],
  },
}
`;
